src/ESP32_VS1053_Stream.cpp

```diff
@@ -417,6 +417,10 @@ void ESP32_VS1053_Stream::setVolume(const uint8_t vol) {
     if (_vs1053 && !_startMute) _vs1053->setVolume(_volume);
 }
 
+void ESP32_VS1053_Stream::setTone(uint8_t *rtone) {
+     _vs1053->setTone(rtone);
```

Comment:
Please check if _vs1053 is not empty before

Reply:
Agree, I am using this currently without any problem. Do you think that there have to be some more checks?

Reply:
I think just checking for a non NULL pointer to the hardware is enough.

```c++
if (_vs1053) _vs1053->setTone(rtone);
```

Reply:
Works for me - feel free to add.